portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -45,23 +56,127 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  @Test 
+  public void testPOSTSingleComment() throws Exception {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+    HttpServletRequest postRequest = mock(HttpServletRequest.class);
+    HttpServletResponse postResponse = mock(HttpServletResponse.class);
```

Comment:
Another thing that would reduce boilerplate here is to refactor objects common to all of the test methods to be test class members. In this case I would do this for:

- DatastoreService
- HttpServletRequest
- HttpServletResponse
- DataSevlet

Reply:
done!